Dockerfile

```diff
@@ -17,3 +17,4 @@ RUN poetry install
 # Copy files to image
 COPY ./src ./src
 COPY ./cli ./cli
+COPY ./data ./data
```

Comment:
Just a note - if the `data` folder will change less frequently that the `src` folder then move it before

Reply:
I imagine it'll change more frequently than `src`, as this is where the input spec for the pipeline goes. Thanks for the tip though 👍